.github/actions/native-models/entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh -e
+
+# Describe dependencies (for debugging Docker)
+git --version
+thrift --version
+
+# Git setup
+export ACCESS_TOKEN=$1
+git config --global credential.helper "/bin/bash /credential-helper.sh"
+git config --global user.email '<>'
+
+# fixes error: "fatal: detected dubious ownership in repository at '/github/workspace'"
+git config --global --add safe.directory /github/workspace
+
+# We get the version and tag from inputs
+export VERSION=$2
+export TAG=$3
+
+# Clone repos
+git clone https://github.com/guardian/mobile-apps-api-models.git
+git clone https://github.com/guardian/mapi-models-swift.git
+
+# Generating the swift models is part of the release flow
+# We should only be releasing after merging to the main branch
+echo "Generating Swift models..."
+thrift --gen swift -r mobile-apps-api-models/models/src/main/thrift/collection.thrift
+
+# Hopefully -f is a first-time-only thing
+rm -f mapi-models-swift/Sources/Collection/*.swift
+cp gen-swift/* mapi-models-swift/Sources/Collection
+
+# Commit and tag latest swift models
+cd mapi-models-swift
+git add Sources/Collection/*.swift
```

Comment:
Here's the updated needed for the package name change
```suggestion
rm -f mapi-models-swift/Sources/Collection/*.swift
rm -f mapi-models-swift/Sources/Blueprint/*.swift
cp gen-swift/* mapi-models-swift/Sources/Blueprint

# Commit and tag latest swift models
cd mapi-models-swift
git add Sources/Blueprint/*.swift
```

Reply:
Thanks Ben - I've updated the package and this workflow accordingly. You mentioned that maybe a release in the swift package would be preferable to just a tag. I've created an [issue](https://github.com/guardian/mobile-apps-api-models/issues/5) to track this, but I'd like to merge this PR sooner rather than later (it's difficult to test/release while this workflow isn't on `main`)

The most recent [release](https://github.com/guardian/mobile-apps-api-models/releases/tag/v0.0.5) of the schema resulted in the [package](https://github.com/guardian/mapi-models-swift/releases/tag/0.0.5) being updated - the `Card` now has the latest properties.